dev/fattest.simplicity/src/componenttest/custom/junit/runner/LogPolice.java

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package componenttest.custom.junit.runner;
+
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.custom.junit.runner.Mode.TestMode;
+
+/**
+ * Used to ensure a single FAT does not produce too much trace logs.
+ */
+public class LogPolice {
+
+    private static final long MB = 1024000L;
+    // TODO: Start out with 4GB cap on trace in LITE mode, incrementally constrict this down to ~300MB trace allowed per bucket
+    private static final long MAX_ALLOWED_TRACE = MB * Integer.getInteger("fat.test.max.allowed.trace.mb", 4000); // 4GB max (for now), override w/ system prop
```

Comment:
Does this delegate eventually to a build prop? Seems like it does, but it's easier to just ask.

Reply:
Yes, the value here is parsed from the `fat.test.max.allowed.trace.mb` system property.  If that property is not set, then a default value of 4000 is used.